tabs/utils/tab_data.toml

```
@@ -12,6 +12,10 @@ script = "bluetooth-control.sh"
 name = "Numlock on Startup"
 script = "numlock.sh"
 
+[[data]]
+name = "Ollama Chatbot"
```

```suggestion
name = "Ollama"
```

Maybe just `Ollama` would be better. Up to you.